src/loginRouter.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
```

Comment:
We have a bunch of unused variables here.

Also, `router` can be done in one line: `const router = require('express').Router()`